content/soarian/dstu2/soarian_authorization.md

```diff
@@ -0,0 +1,119 @@
+In order to access data from Soarian via the API, an Application has to first acquire an access token from the API's authorization server. For detailed information on OAuth 2.0 refer to [Authorization] [authorization-OAuth]
+
+* TOC
+{:toc}
+
+## Requesting Authorization on Behalf of a User
+
+To access Cerner's FHIR<sup>&reg;</sup> services on behalf of a user, your client Application must make an authorization request through a user agent on the user's device. This involves the following steps:
+
+  * Discover the authorization and token endpoint URLs.   
+  * For Soarian FHIR Conformance details refer to [Conformance] [conformance]  
+  * Construct an authorization grant request URL.  
+  * Invoke a browser with the authorization grant request URL.  
+  * Receive the OAuth callback.  
+  * Exchange the received grant for a token.  
+  * Close the browser.  
+  * Handle any exceptions encountered.  
+  * Utilize "refresh" tokens.  
+  
+For details on above steps refer to [Authorization] [authorization-behalf-user]
+Soarian supports only SMART Application Launch flow as listed in the [technical specification] [smart-application]
+Soarian supports only "patient" scope for the resources specified in the Conformance statement. (For example: patient/Condition.read, patient/MedicationStatement.read etc.)
+
+## ActivateLinkToken API to link User Identity to Patient
+
+### Overview
+
+Soarian Clinicals generates and provides Activation tokens for each of the patients and/or patient's proxies who wish to request patient's data and the Application needs to provide a means to enter Activation tokens and call ActivateLinkToken API to link the user to the patient record (user is linked as a patient or a defined patient proxy).  
+
+Activation tokens can be provided to patients and their proxies. Soarian Clinicals allows hospital staff to add patient proxies and manage their tokens. This Activation token is provided to patient and/or their proxies by hospital staff either by printed report or email, based on hospital's configuration and policy.  Activation tokens are temporary and expire if not utilized; the API will error if a token is invalid, revoked or expired.  Patients can request hospital staff to generate another Activation token, if needed.  
+
+API-consuming Application(s) need to invoke ActivateLinkToken API to establish a link between the established user identity and a patient record in Soarian Clinicals. This service when called, the Application must provide Authorization token (access token received from Cerner OAuth server), valid Activation token (provided by Hospital to Patient or Proxy) and Patient's Date of Birth. This ActivateLinkToken API to link the user with a patient record is a one-time action and must be called prior to making any API calls for other FHIR resources supported by Cerner Soarian Clinicals.  The user to patient link established remains valid until the time it is manually revoked by hospital staff on patient request. 
```

Comment:
Activation is once per account (patient/proxy), regardless of the number of different applications used to access the patient record.  @slvsridhar will clarify in the documentation.